Quadriga/src/main/java/edu/asu/spring/quadriga/rest/open/NodeInfoController.java

```diff
@@ -111,4 +112,12 @@ public int compare(TextPhrase o1, TextPhrase o2) {
         return new ResponseEntity<String>(result.toString(), HttpStatus.OK);
 
     }
+
+    public void jsonKeyValueAppend(JSONObject occurance, TextOccurance occur) throws JSONException {
```

Comment:
this still doesn't say what exactly this method does. name it something like addTextInfo or appendTextInformation